File: api/RS.API/RS.Entity/Models/UserRoles.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace RS.Entity.Models
{
    public partial class UserRoles : BaseEntity
    {
        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public Guid UserRolesId { get; set; }

        public Users user { get; set; }

        public Guid UserId { get; set; }

        public int RoleId { get; set; }

        public Roles Role { get; set; }
    }
}

File: web/src/app/webapi/models/user-roles.ts
/* tslint:disable */
import { Roles } from './roles';
import { Users } from './users';

/**
 */
export class UserRoles {
    createdBy?: string;
    userRolesId?: string;
    userId?: string;
    roleId?: number;
    role?: Roles;
    isActive?: boolean;
    user?: Users;
    createdDate?: string;
    modifiedBy?: string;
    modifiedDate?: string;
    isDeleted?: boolean;
    deletedBy?: string;
    deletedDate?: string;
}
